docs/source/routing/performance/caching/entity.mdx

```diff
@@ -61,6 +61,86 @@ With entity caching enabled for this example, the router can:
 - Cache the cart per user, with a small amount of data.
 - Cache inventory data with a short TTL or not cache it at all.
 
+```mermaid
+---
+title: Entity Caching using the "Price" entity
+---
+flowchart RL
+	subgraph QueryResponse["JSON Response"]
+		n1["{
+			&emsp;&emsp;&quot;products&quot;: [
```

Comment:
I had trouble with spacing out the content in these code blocks, as the Markdown support within Mermaid blocks is minimal, so I leaned on the HTML character encoding you see here.